static/src/javascripts/projects/commercial/modules/article-aside-adverts.js

```diff
@@ -46,18 +49,37 @@ export const init = (start: () => void, stop: () => void): Promise<boolean> => {
             ]
         )
         .then(([mainColHeight, immersiveOffset]: [number, number]) => {
+            if (config.get('page.hasShowcaseMainElement', false)) {
+                const slotWrapper = createSlotWrapper();
+                const asideSlots = createSlots('right-with-showcase', {
+                    classes: 'mpu-banner-ad',
+                });
+
+                asideSlots.forEach(adSlot => {
+                    slotWrapper.append(adSlot);
+                });
+                return fastdom
+                    .write(() => {
+                        $col.prepend(slotWrapper);
+                    })
+                    .then(() => {
+                        addSlot(asideSlots[0], false);
```

Comment:
My fastdom knowledge is largely lost to the annals of time and I guess deep somewhere in `addSlot` maybe there's a write or read but I *think* maybe these could be in the write?